more pandas and zoos in Europe

akapanredpanda opened the pull request:
References with a ton of photos. The URLs are from Facebook because it's easier to search for information than Instagram ([Auvergne](https://www.instagram.com/parcanimalierdauvergne/), [Haute-Touche/MNHN](https://www.instagram.com/le_museum/), [Zoom Torino](https://www.instagram.com/zoomtorino/), [Hellabrunn](https://www.instagram.com/tierpark_hellabrunn/), [Zoo Santo Inácio](https://www.instagram.com/zoo_santoinacio/)):
https://www.facebook.com/ParcAnimalierdAuvergne/posts/1809457942565077
https://www.facebook.com/ParcAnimalierdAuvergne/posts/1611368982373975
https://www.facebook.com/ParcAnimalierdAuvergne/posts/1621496078027932
https://www.facebook.com/ParcAnimalierdAuvergne/posts/1574788332698707
https://www.facebook.com/ParcAnimalierdAuvergne/posts/1559709530873254
https://www.facebook.com/ParcAnimalierdAuvergne/posts/1467117566799118
https://www.facebook.com/ParcAnimalierdAuvergne/posts/1222357897941754
https://www.facebook.com/ParcAnimalierdAuvergne/posts/1133896193454592
https://www.facebook.com/ParcAnimalierdAuvergne/posts/1090074571170088
https://www.facebook.com/ParcAnimalierdAuvergne/posts/425314324312786
https://www.facebook.com/ParcAnimalierdAuvergne/posts/1635247796652760
https://www.facebook.com/ReserveHauteTouche/photos/a.1624856437596259/1624856484262921/
https://www.facebook.com/ReserveHauteTouche/posts/1454214877993750
https://www.facebook.com/ReserveHauteTouche/posts/1405687112846527
https://www.facebook.com/ReserveHauteTouche/posts/3088407671241121
https://www.facebook.com/ReserveHauteTouche/posts/1987320268016539
https://www.facebook.com/zoomtorino/posts/10161284431000478
https://www.facebook.com/zoomtorino/posts/10162973490410478
https://www.facebook.com/zoomtorino/photos/a.282880175477/10158184098895478/
https://www.facebook.com/zoomtorino/posts/10155444081925478
https://www.zoo.pt/pt/visitar/novidades/no-dia-internacional-do-panda-vermelho-o-jardim-zoologico-apresenta-a-nova-cria/
https://sicnoticias.pt/pais/2019-07-23-Dois-pandas-vermelhos-recebidos-no-Zoo-de-Santo-Inacio
https://www.hellabrunn.de/en/the-latest-news/news/red-panda-shamina-gets-a-new-home/182bc939e228ed833eaf2230d2adfa4e/news/
https://www.hellabrunn.de/en/the-latest-news/news/welcome-justin-new-red-panda-at-hellabrunn/c9c3300f496a72b29ab4e934c1eaf509/news/
https://www.hellabrunn.de/en/the-latest-news/news/red-panda-comes-out-of-hiding/5d0bbedfaa3de6616cbd6672dd43a29e/news/
https://www.hellabrunn.de/en/the-latest-news/news/red-panda-day-at-hellabrunn-zoo-this-saturday/8ea097d8e2ea1c85335b82912cc8380f/news/
https://www.hellabrunn.de/en/the-latest-news/news/baby-red-panda-is-a-girl/2e92ec1456c486c79d050a6b35277a45/news/
https://www.facebook.com/tierparkhellabrunn/posts/2127246263976304
https://www.hellabrunn.de/en/the-latest-news/news/hellabrunns-baby-red-panda-gets-first-medical-check-up/f99dc72d62134ab8c67984a640588cc9/news/
https://www.hellabrunn.de/en/the-latest-news/news/it-s-a-boy-hellabrunn-zoo-s-red-pandas-give-birth-again/6a75414b326eb0c41bb3bd414cc862e6/news/
https://www.facebook.com/tierparkhellabrunn/posts/3624274784273437
https://www.facebook.com/tierparkhellabrunn/posts/2795818327119091
http://www.hellabrunn.de/neues-aus-hellabrunn/newsanzeige/willkommen-justin-neuer-roter-panda-in-hellabrunn/ec9f7a6d71392a0a105f2d19e6d258ab/aktuelles/
https://www.facebook.com/tierparkhellabrunn/photos/a.400221913345423/1471679999532937/
https://www.facebook.com/tierparkhellabrunn/posts/2613644005336525
https://www.facebook.com/tierparkhellabrunn/posts/3466059076761676
https://www.facebook.com/tierparkhellabrunn/posts/1223931120974494
https://www.researchgate.net/profile/Jorge-Correia-12/publication/341279799_Carcinoma_bronquiolo-alveolar_e_infeccao_por_Angiostrongylus_vasorum_num_panda_vermelho_Ailurus_fulgens_fulgens/links/5eb81fc44585152169c14f66/Carcinoma-bronquiolo-alveolar-e-infeccao-por-Angiostrongylus-vasorum-num-panda-vermelho-Ailurus-fulgens-fulgens.pdf
https://www.presstur.com/destinos/ca-dentro/natal-chega-ao-jardim-zoologico-com-duas-crias-de-panda-vermelho-4/
https://www.cmjornal.pt/multimedia/videos/detalhe/panda-vermelho-bebe-e-o-novo-residente-do-jardim-zoologico-de-lisboa

wwoast commented:
Thanks for submitting this! I've been to Hellabrunn so I'm grateful to include those animals, especially since I share a name with one!

I see some typoes in this PR still. The `build.py` script [stops when it finds a problem](
https://travis-ci.com/github/wwoast/redpanda-lineage/builds/222965007) -- but I saw others (one of your files has duplicate `location.2` fields). If you don't have a chance to fix these I'll try and get these fixed/merged later tonight.

Justin

akapanredpanda commented:
You're welcome!
I didn't notice the panda and you have the same name :D
I've fixed many typoes and added the [first location date](https://www.hellabrunn.de/en/the-latest-news/news/welcome-justin-new-red-panda-at-hellabrunn/c9c3300f496a72b29ab4e934c1eaf509/news/) of Justin.
